pom.xml

```
@@ -279,6 +278,18 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jbpm-installer-exclude-productized</id>
```

Why not using same id profile as per jbpm-distribution/pom.xml in line 404 
```suggestion
            <id>jbpm-installer-include</id>
```